pages/docs/tutorials/generate-code.md

```diff
@@ -0,0 +1,136 @@
+
+1. Trigger generation of the Node.js code:
+
+<CodeBlock language="bash">
+{`asyncapi generate fromTemplate asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto`}
+</CodeBlock>
+
+From the command above;
+
+- `asyncapi generate fromTemplate` is how you use AsyncAPI Generator through the AsyncAPI CLI. 
+
+- ` asyncapi.yaml` is how you point to your AsyncAPI document. It can also be a URL. 
+
+- `@asyncapi/nodejs-template` is how you specify the Node.js template.
+ 
+- `-o` determines where to output the result.
+
+- `-p` defines additional parameters you want to pass to the template, in this instance, the `server` parameter which is where you specify the server's name as it is defined in AsyncAPI document.
+
+2. List all files in directory and check that Node.js application is generated:
+<CodeBlock language="bash">
+{`cd output && ls`}
+</CodeBlock>
+
+Upon execution of the command above, the following is an example of the expected result;
+```
+$ ls
+Dockerfile
+asyncapi.yaml
+docs
+src
+README.md
+config
+package.json
+```
+
+
+## Start generated application
+
+1. Install dependencies of newly generated application:
+<CodeBlock language="bash">
+{`npm install`}
+</CodeBlock>
+
+2. Start the application:
+<CodeBlock language="bash">
+{`npm start`}
+</CodeBlock>
+
+## Send a message to the broker
+
+1. In another terminal install the MQTT.js library:
+<CodeBlock language="bash">
+{`npm install mqtt -g`}
+</CodeBlock>
+
+2. Send a message to MQTT broker that the generated application is now connected to. You will use the MQTT client command in your terminal:
+<CodeBlock language="bash">
+{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": 3, "sentAt": "2017-06-07T12:34:32.000Z"}'`}
+</CodeBlock>
+
+3. Go back to the previous terminal and check if your application logged the streetlight condition you just sent. You should see something like this displayed in the terminal;
+
+```
+light/measured was received:
+{ id: 1, lumens: 3, sentAt: '2017-06-07T12:34:32.000Z' }
+```
+
+# Summary
+
+In this tutorial, you learned how to generate your code from the Streetlights API specification document created in the [previous tutorial]() using the AsyncAPI generator tool. 
+
+Additionally, you have learned how to run your code by installing the generated code's dependencies and sending several test messages to the Streelights application using the MQTT client.
+
+
+# Next steps
+Now that you've completed this tutorial, go ahead to learn how to [validate your document using the AsyncAPI studio]() through the validation techniques supported by AsyncAPI.
+
+---
+
+<DocsButton
+  suggestions={[
+    {
+      href: '/docs/tutorials',
+      title: 'Tutorials - Overview',
+      type:'back',
+    },
+    {
+      href: '/docs/tutorials/streetlights-interactive',
+      title: 'Streetlights - Interactive (Alpha)',
+      type:'next',
+    }
+  ]}
+/>
```

Comment:
Hey @Annysah, this is wrong. 😄✌🏽 Why do you have prev/next buttons that link to a streetlights tutorial that we are removing? Remember how this PR replaces the streetlights tutorial? Second, this tutorial will **not** go after the `Tutorials Overview` page, this tutorial is the 3rd item in line. 

Remember, the 4 new tutorials to replace Streetlights tutorial will go in this order:
1. create AsyncAPI document
2. validate AsyncAPI document
3. generate code
4. validate messages/events 

**Please update this PR and fix the prev/next buttons to link to the correct order of tutorials.**

Reply:
oh yes, I left it because I was uncertain about how to link it correctly.
will this be a right link for the prev button '/docs/tutorials/validate-documents.md' and '/docs/tutorials/message-validation.md' for the next button?

Reply:
yes, exactly right @Annysah 😄✌🏽 

go forth and update the buttons 🙌🏽🙌🏽🙌🏽

Reply:
Thank you!...updating right away